pkg/cmd/cli/schedule/create.go

```diff
@@ -95,7 +95,7 @@ func NewCreateOptions() *CreateOptions {
 func (o *CreateOptions) BindFlags(flags *pflag.FlagSet) {
 	o.BackupOptions.BindFlags(flags)
 	flags.StringVar(&o.Schedule, "schedule", o.Schedule, "A cron expression specifying a recurring schedule for this backup to run")
-	flags.BoolVar(&o.UseOwnerReferencesInBackup, "use-owner-references-in-backup", o.UseOwnerReferencesInBackup, "Specifies whether to use OwnerReferences on backups created by this Schedule")
+	flags.BoolVar(&o.UseOwnerReferencesInBackup, "use-owner-references-in-backup", o.UseOwnerReferencesInBackup, "Specifies whether to use OwnerReferences on backups created by this Schedule. Notice: if set to true, when schedule is deleted, backups will be deleted too. Details please reference to https://github.com/vmware-tanzu/velero/issues/4093")
```

Comment:
I'm considering should we add the wording `Details please reference to https://github.com/vmware-tanzu/velero/issues/4093"` to the helper message or not.

Reply:
The information should be placed some where, and I'm not sure cli information is best place too. 
How about remove the document part from here, then put them in the site's backup schedule part?

Reply:
Sounds good to me.